plugins/kubernetes/app/controllers/kubernetes/deploy_group_roles_controller.rb

```diff
@@ -88,7 +88,7 @@ def seed
         errors = ["Roles failed to seed, fill them in manually."]
         errors.concat(created.map do |dgr|
           "#{dgr.kubernetes_role.name} for #{dgr.deploy_group.name}: #{dgr.errors.full_messages.to_sentence}"
-        end)
+        end.first(3))
```

Comment:
Maybe add a '...' or something to indicate the messages are being truncated if there are more than 3?